### PR TITLE
Config UI

### PR DESCRIPTION
**Pull request by KeeyanGhoreshi**
Adds a collection for the config.  There's no additional UI made for this, since the config is a collection and is being displayed as a regular collection.  The only difference is that you're not allowed to delete config items.  This was built on top of the config branch, so you should be able to see the default contents of the config file in the collection.

### CODE REVIEW

File: src/ui/components/Collections/Collections.jsx [hunk 1]
```diff
@@ -101,9 +101,8 @@ const Collections = props => {
           { value: 'secret', label: 'secret', edit: true },
           // NOTE: secret is only used for client_secret_basic auth.
           // see src/utils/client.js : connectToServer for more info
-          { value: 'customScopes', label: 'customScopes', edit: true },
-          { value: 'type', label: 'type', edit: false }
-        ];
+          { value: 'customScopes', label: 'customScopes', edit: true }
+]        ];
```
Comment (blangley28) [inline src/ui/components/Collections/Collections.jsx:105]: Extra `]` here
```suggestion
        ];
```

**Review by blangley28 — COMMENTED** [review]
Besides that one syntax error this works well. Functionally it does everything we need to but it would be nice if it looked more like a settings menu (i.e. toggles for the true/false values like this https://material-ui.com/components/switches/) instead of just the generic collections screen. 
I know we don't have designs but we could come up with something simple. @dehall what do you think? Or is the ability to edit them enough?

File: src/ui/components/Collections/Collections.jsx (hunk 1 above)
Comment (KeeyanGhoreshi) [reply src/ui/components/Collections/Collections.jsx:105]: i wonder why this didn't produce an error and the linter didn't say anything

**Comment by dehall** [comment]
Agreed a more targeted UI would be nice, but I think the ability to edit them is enough for now. Maybe Noranda has some existing designs for settings pages, or we can ask her for something new, and we can make that a new task. This might be useful during the connectathon so I don't want to hold it up too long

**Pull request merged**